sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -40,7 +40,8 @@ export class CheckpointManager {
 // @public
 export enum CloseReason {
     OwnershipLost = "OwnershipLost",
-    Shutdown = "Shutdown"
+    Shutdown = "Shutdown",
+    Unknown = "Unknown"
```

Comment:
We don't have the `Unknown` reason anymore in the master branch. Please merge/rebase from master and run build again.

Reply:
`rush build -t @azure/core-http` is failing on merge from master..